doc/contributing/maintaining/maintaining-dependencies.md

```diff
@@ -189,7 +189,7 @@ In practical terms, Corepack will let you use Yarn and pnpm without having to
 install them - just like what currently happens with npm, which is shipped
 by Node.js by default.
 
-### googletest
+### googletest ec4fed9
```

Comment:
This is a bit weird, it's the only one in the list that uses a hash as version number..